WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalScanningForReader.swift

```diff
@@ -21,16 +21,27 @@ final class CardPresentModalScanningForReader: CardPresentPaymentsModalViewModel
 
     let auxiliaryButtonTitle: String? = nil
 
+    let auxiliaryButtonimage: UIImage? = .infoOutlineImage
+
     var auxiliaryAttributedButtonTitle: NSAttributedString? {
-        let moreLinkAttributes: [NSAttributedString.Key: Any] = [
-            .foregroundColor: UIColor.accent,
-            .underlineStyle: NSUnderlineStyle.single.rawValue
-        ]
-        let moreTextAttributes: [NSAttributedString.Key: Any] = [.foregroundColor: UIColor.text]
-        let linkAttributedString = NSMutableAttributedString(string: Localization.learnMoreLink, attributes: moreLinkAttributes)
-        let moreAttributedString = NSMutableAttributedString(string: Localization.learnMoreText, attributes: moreTextAttributes)
-        linkAttributedString.append(moreAttributedString)
-        return linkAttributedString
+        let result = NSMutableAttributedString(
+            string: .localizedStringWithFormat(
+                Localization.learnMoreText,
+                Localization.learnMoreLink
+            ),
+            attributes: [.foregroundColor: UIColor.text]
+        )
+        result.replaceFirstOccurrence(
+            of: Localization.learnMoreLink,
+            with: NSAttributedString(
+                string: Localization.learnMoreLink,
+                attributes: [
+                    .foregroundColor: UIColor.accent,
+                    .underlineStyle: NSUnderlineStyle.single.rawValue
+                ]
+            ))
+        result.addAttribute(.font, value: UIFont.footnote, range: NSRange(location: 0, length: result.length))
+        return result
```

Comment:
In my next PR there will be a `LearnMoreViewModel`, which you may want to use to produce this string. It's just extracted from the SwiftUI view. Can be in a later PR, it would be best to get this merged 😊

Reply:
Nice! I'll keep an eye on it :D